packages/nimble-components/src/patterns/button/styles.ts

```diff
@@ -207,26 +220,38 @@ export const styles = css`
         css`
             .control {
                 background-color: rgba(${borderRgbPartialColor}, 0.1);
-                border-color: transparent;
             }
 
             .control:hover {
                 background-color: transparent;
-                border-color: ${borderHoverColor};
+                background-image: linear-gradient(
+                    rgba(${borderRgbPartialColor}, 0.1),
+                    rgba(${borderRgbPartialColor}, 0.1)
+                );
+                background-size: calc(100% - 4px) calc(100% - 4px);
+                background-repeat: no-repeat;
+                background-position: center;
```

Comment:
I find the animation to be really jarring. When just using the control at a normal size it feels like it's glitching. I think it's because the space disappears immediately around the block instead of shrinking and then the border grows:
![hoveranimation](https://github.com/ni/nimble/assets/1588923/07fdad33-5850-49d3-a1e9-b2039dcdf6ae)

If the background-size animates in it feels a lot smoother and less jumpy:

![hoveranimation2](https://github.com/ni/nimble/assets/1588923/3c673c37-465c-4fa9-902a-5b99601f179a)

Could be done with something like this:

![image](https://github.com/ni/nimble/assets/1588923/cfdf0caa-5fb0-4581-83bc-61dac249454a)